staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -671,6 +674,13 @@ func (s *Controller) getNodeConditionPredicate() NodeConditionPredicate {
 			return false
 		}
 
+		// Remove nodes that are about to be deleted by the cluster autoscaler.
+		for _, taint := range node.Spec.Taints {
+			if taint.Key == ToBeDeletedTaint {
```

Comment:
Can we add a V5 Info log that we failed node predicate because its about to be deleted by the cluster autoscaler.

Reply:
Done